README.md

```diff
@@ -96,52 +81,42 @@
   * 如果 review 完成，但还需要一些人工验证的话，为避免过早自动合并，可以通过评论命令来阻碍
     * 例如：通过评论 `/hold` 来阻止机器人账号自动合并
 
-## 社区运营
+## 社区治理
+
+随着社区规模的变大，社区治理的重要性会日益凸显。可能遇到的问题包括如下：
+
+* 如何帮助新人快速地熟悉、适应社区的氛围、玩法
+* 利用适当的社媒、多媒体平台宣传社区理念、技术，吸引有相似兴趣的人加入
+* 结合社区成员自身的特点、发展诉求，通过复盘、再组织等方法，适时地调整社区结构
+* 从合规（许可证选用、商标、专利等）层面确保社区的利益
+* 社区基础设施的维护
+* 设置沟通、交流底线（或规则，code-of-conduct），避免不好的行为、言论
 
 对于社区（Community）这个词，不同的人有着不同的理解。本文的社区专指：开源社区（Open Source Community）。
 
-在社区运营过程中，有很多好的实践和方法，例如：社交媒体、Meetup、SIG、TOC 等的运营。下面，给出部分组织形式的最佳实践指导。
-
-### SIG 运营
-
-确定 SIG 的准入准出制度，根据团队的实际情况来控制 SIG 的数量，运营的质量是首要考虑的。
-
-SIG 的健康发展，离不开公开、透明、多元化等原则，流于形式的、不符合开源精神的 SIG 没有存在的意义。
-
-### SIG 会议
-
-* 会议时间的选择，尽量考虑多数人的需求
-  * 通过 [Doodle](https://doodle.com/en/) 或类似的工具供大家投票选择
-* 每次会议都要有会议开始和结束的表述
-  * 例如：现在开始我们今天的例会；结束时可以说：“非常感谢各位参加今天的社区例会”等等的
-* 热情地欢迎首次参加例会的成员，并鼓励做自我介绍
-* 做必要的上下文介绍
-  * 不管是拨入会议还是观看回放的人，都有可能是首次参与，缺少上下文会导致他们难以明白相关讨论
-  * 如果有文字性的上下文介绍，尽量在会议记录文档中有体现
-* 主持人要把控会议时间、节奏
-  * 尽量控制在 1 小时内，未讨论完的议题放到下次讨论
-  * 在讨论过程中帮助、引导得出结论
-* 使用适当的表述方式
-  * 避免使用类似于“我们昨天讨论过xxx”的表述，社区成员无法了解到你们讨论的内容
-* 确保例会的周期性
-  * 通常情况下，每两周一次的例会比较适中
-  * 当参与者人数相对较多，而且分布在不同时区的话，结合参与者的意见可以分两个会议
-  * 当可以主持会议的人都无法参加，或者其他需要取消例会的情况，尽早地通知社区会议取消
-* 会议记录
-  * 文字记录，可以使用 [Google Document](https://docs.google.com/document/u/0/) 或者[腾讯文档](https://docs.qq.com/)
-  * 会议录屏，根据会议参与者使用的语言不同，中文可以选择[哔哩哔哩](https://www.bilibili.com/)，英文可以选择 [YouTube](https://www.youtube.com/)
+在社区运营过程中，有很多好的实践和方法，例如：社交媒体、Meetup、SIG、Technical Oversight Committee (TOC) 等的运营。下面，给出部分组织形式的最佳实践指导。
+
+### SIG
+
+[Special Interest Group，特别兴趣小组](sig.md)，简称 SIG。在大中型开源社区中，这是一种便于社区成员在特定领域交流、讨论，更加聚焦地解决相关问题的组织形式。
+
+开源社区中，可以根据不同的领域设立相应的 SIG。每个 SIG 往往会有一名或多名 lead（领队），以及多名活跃的社区成员作为 SIG 的成员（member）。在不同的社区中，领队可能会有不同的叫法，例如：co-chair、co-lead 等。值得注意的是，这里所使用的表述是——“领队”，而不是领导、经理（manager）、管理员（admin）、负责人；领队的重要意义在于维护 SIG 的积极、活跃，而不是特权。
```

Comment:
```suggestion
开源社区中，可以根据不同的领域设立相应的 SIG。每个 SIG 往往会有一名或多名 leader（领队），以及多名活跃的社区成员作为 SIG 的成员（member）。在不同的社区中，领队可能会有不同的叫法，例如：co-chair、co-lead 等。值得注意的是，这里所使用的表述是——“领队”，而不是领导、经理（manager）、管理员（admin）、负责人；领队的重要意义在于维护 SIG 的积极、活跃，而不是特权。
```

Reply:
关于使用 leader 还是 lead，我也有点困惑，例如：下面就使用的是 lead
https://github.com/cdfoundation/toc/blob/acfdc75044170517c1118208e81411e561e5b4b5/sigs/sig-mlops.md

Reply:
我搜索到如下的区别分析：

```
A “leader” is a person who is acting in a role of authority [over some group]. “Lead” can be an adjective meaning “first” or “front” among a group, but it can be a noun meaning “person or persons that have the role of going in front or holding authority in a group.”
```

Reply:
这个我也不太懂，我以为形容「人」就要加上 `er`。但是根据提供的信息来看 `lead` 的使用是没有问题的。